New doc: Manage clusters (#886)
astronomer/docs: astro/modify-cluster.md

- A cluster on Astro.
- Permission from your team.

> If you don't have a cluster on Astro, follow the instructions to [Install Astro on AWS](install-aws.md) or [GCP](install-gcp.md). If you have an existing cluster and are interested in creating additional clusters, read [Create a cluster](create-cluster.md). To view your clusters' current configurations, see [Manage clusters](manage-clusters.md).

Reviewer: ```suggestion
If you don't have a cluster on Astro, see [Install Astro on AWS](install-aws.md) or [GCP](install-gcp.md). If you have an existing cluster and you want to create additional clusters, see [Create a cluster](create-cluster.md). To view your clusters' current configurations, see [Manage clusters](manage-clusters.md).
```
